runtime/src/lib.rs

```diff
@@ -52,11 +53,11 @@ pub use primitives_core::{
 };
 
 pub mod constants;
-use self::constants::time::*;
```

Comment:
Start from edition 2018, we recommend adding `self`/`super`/`crate` to all imports except external crate.